tests/catch/unit/stream/hipStreamGetFlags.cc

```diff
@@ -27,52 +27,43 @@ Testcase Scenarios :
 #include <hip_test_common.hh>
 
 /**
- * Test flag value of stream created with various types.
+ * @brief Check that hipStreamGetFlags returns the same flags that were used to create the stream.
+ *
  */
-TEST_CASE("Unit_hipStreamGetFlags_BasicFunctionalities") {
+TEST_CASE("Unit_hipStreamGetFlags_Basic") {
+  unsigned int expectedFlag = GENERATE(hipStreamDefault, hipStreamNonBlocking);
+  unsigned int returnedFlags;
   hipStream_t stream;
-  unsigned int flags;
-  // Check flag value of stream created with hipStreamCreateWithFlags
-  SECTION("Check flag value of streams hipStreamCreateWithFlags") {
-    HIP_CHECK(hipStreamCreateWithFlags(&stream, hipStreamDefault));
-    HIP_CHECK(hipStreamGetFlags(stream, &flags));
-    REQUIRE(flags == hipStreamDefault);
-    HIP_CHECK(hipStreamDestroy(stream));
-    HIP_CHECK(hipStreamCreateWithFlags(&stream, hipStreamNonBlocking));
-    HIP_CHECK(hipStreamGetFlags(stream, &flags));
-    REQUIRE(flags == hipStreamNonBlocking);
-    HIP_CHECK(hipStreamDestroy(stream));
-  }
-  // Check flag value of stream created with hipStreamCreate
-  SECTION("Check flag value of streams hipStreamCreate") {
-    HIP_CHECK(hipStreamCreate(&stream));
-    HIP_CHECK(hipStreamGetFlags(stream, &flags));
-    REQUIRE(flags == hipStreamDefault);
```

Comment:
this should still be checked

Reply:
This tests have been moved to hipStreamCreate.cc file as part of: https://github.com/ROCm-Developer-Tools/HIP/pull/2560/commits
It's the function call in line 29. That PR is already merged.